$OpenBSD: patch-libs_gst_helpers_gst-ptp-helper_c,v 1.1 2015/09/27 08:21:39 ajacoutot Exp $

XXX push upstream
/usr/include/net/if.h:351: error: field 'ifru_addr' has incomplete type

--- libs/gst/helpers/gst-ptp-helper.c.orig	Sat Sep 26 09:17:14 2015
+++ libs/gst/helpers/gst-ptp-helper.c	Sat Sep 26 09:20:26 2015
@@ -37,6 +37,7 @@
 #include <sys/types.h>
 #include <errno.h>
 #include <sys/ioctl.h>
+#include <sys/socket.h>
 #include <net/if.h>
 #include <netinet/in.h>
 #include <string.h>
